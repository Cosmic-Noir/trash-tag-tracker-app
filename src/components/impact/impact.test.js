import React from "react";
import ReactDom from "react-dom";
import renderer from "react-test-renderer";
import Impact from "./impact";

describe("Impact Component", () => {
  it("Renders without crashing", () => {
    const div = document.createElement("div");
    ReactDom.render(<Impact />, div);
    ReactDom.unmountComponentAtNode(div);
  });
  it("Renders the UI as expected", () => {
    const tree = renderer.create(<Impact />).toJSON;
    expect(tree).toMatchSnapshot();
  });
});
