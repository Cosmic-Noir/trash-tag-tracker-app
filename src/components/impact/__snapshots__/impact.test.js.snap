// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Impact Component Renders the UI as expected 1`] = `[Function]`;
